Updated CHANGELOG and necessary files for 3.1.0-alpha.0 release

robinmolen opened the pull request:
Partly closes open-formulieren/open-forms#5102

vaszig commented on CHANGELOG.rst:
```diff
@@ -2,6 +2,49 @@
+Bugfixes
+--------
+
+* [#4510] Fixed generic and user-unfriendly error messages in the submission summary.
```
I think this has to do with the formstep as well where the backend errors were not showed properly.

robinmolen replied:
For this fix only the submission summary components have been changed, so i think it's only a summary view fix?

vaszig reviewed (changes requested):
I have added some comments/recommendations but as I have understood from my personal experience, Changelog is difficult.. 

Maybe we should also add a description about the following to the changelog, what do you think?

- Errors in ErrorBoundary are captured with Sentry.
- Enabled Codecov JS bundle analysis.

And I see some missing translations :)

robinmolen commented on CHANGELOG.rst:
```diff
@@ -2,6 +2,49 @@
+
+* [#5033] User-friendly messages for "Referentielijsten" errors.
+* [#76] Dynamically loading i18n, lodash, map and appointment modules.
+* [#2177] The map component now supports placing a marker, line or polygon.
```
Yeah maybe some more context would be nice.. 👍

robinmolen commented on CHANGELOG.rst:
```diff
@@ -2,6 +2,49 @@
+-------------------
+
+* Removed unused Sentry tracing
+* [#4929] Upgraded project routing
```
I've included these in the routing changes list and updated the title to "Restructured and upgraded React routing".

robinmolen commented:
> I have added some comments/recommendations but as I have understood from my personal experience, Changelog is difficult..

Yeah I totally agree 😄

> 
> Maybe we should also add a description about the following to the changelog, what do you think?
> 
> * Errors in ErrorBoundary are captured with Sentry.
> * Enabled Codecov JS bundle analysis.

Yeah sounds good! I've included these to the "Project maintenance" changes.

> And I see some missing translations :)

Ah nice catch! I've added them 👍

viktorvanwijk commented on src/i18n/messages/nl.json:
```diff
@@ -374,7 +374,7 @@
   },
   "FKWQuw": {
-    "defaultMessage": "Unfortunately, this form is currently unavailable due to an outage. Please try again later.",
+    "defaultMessage": "Helaas is dit formulier op het moment niet te gebruiken door een storing. Probeer het later nog eens.",
```
```suggestion
    "defaultMessage": "Helaas is dit formulier momenteel niet beschikbaar door een storing. Probeer het later nog eens.",
```
Maybe like this?

robinmolen replied:
Yeah sounds good 👍

viktorvanwijk commented on src/i18n/messages/nl.json:
```diff
@@ -1112,7 +1112,7 @@
   },
   "rWbce4": {
-    "defaultMessage": "Form unavailable",
+    "defaultMessage": "Formulier momenteel onbeschikbaar",
```
This is probably really a personal preference, but I would use 'niet beschikbaar'. It's not really consistent elsewhere, though. A quick search reveals that both 'onbeschikbaar'  and 'niet beschikbaar' occur once hehe.

And perhaps without 'momenteel', as there is no 'currently' in English either?

robinmolen replied:
Yeah, "Formulier niet beschikbaar" looks better, and indeed more consistent to the English version.

viktorvanwijk commented on src/i18n/compiled/nl.json:
```diff
@@ -2442,7 +2442,7 @@
     {
       "type": 0,
-      "value": "Klik om een lijk te tekenen"
+      "value": "Klik om een lijn te tekenen"
```
I think we should keep this typo :skull:

viktorvanwijk reviewed (changes requested):
Some minor things :). Mostly personal preference, as there isn't really anything objectively wrong I would say.
